apps/src/netsim/NetSimLobby.js

```diff
@@ -215,86 +273,149 @@ NetSimLobby.prototype.refreshShardList_ = function () {
   });
 };
 
+NetSimLobby.prototype.useRandomShard = function () {
+  this.randomShardID = 'netsim_' + utils.createUuid();
+  this.useShard(this.randomShardID);
```

Comment:
does this.randomShardID get used elsewhere? i dont see where. could it just be a local?